config/initializers/flipper.rb

```diff
@@ -42,10 +42,13 @@
 Flipper::UI.configuration.feature_creation_enabled = false
 # Make sure that each feature we reference in code is present in the UI, as long as we have a Database already
 FLIPPER_FEATURE_CONFIG['features'].each_key do |feature|
```

Comment:
```suggestion
FLIPPER_FEATURE_CONFIG['features'].each do |feature, feature_config|
```
if you change this line you can get rid of line 45